src/main/host/thread_ptrace.c

```diff
@@ -56,6 +56,13 @@ OPTION_EXPERIMENTAL_ENTRY("disable-o-n-waitpid-workarounds", 0, G_OPTION_FLAG_RE
                           "otherwise result in excessive detaching and reattaching",
                           NULL)
 
+static bool _useBufferedIo = false;
```

Comment:
Steve just renamed a bunch of these `_useXXX` switch variables, so maybe we should follow his pattern?

https://github.com/shadow/shadow/pull/1287/files#diff-458bdb269062900e2d0ac153268cbd70b89d5c811f1fd9445d2e49f21c4b5033R55

Although I think he was just aiming to make it so that the default for all of the options is `false`. Anyway, I guess `_enableBufferedIo` would be more consistent with his change?

Reply:
Good point; done

Reply:
> We decided to remove the --disable and --enable prefixes from options, so this PR isn't needed anymore.

I guess I should still use the `--enable` prefix here, and they'll all be changed in another PR?

Reply:
I closed that PR since I thought we wanted to not use the `--enable` or `--disable` prefixes anymore.